src/platform/silabs/wifi/SiWx/WifiInterface.cpp

```diff
@@ -724,14 +657,9 @@ void sl_matter_wifi_post_event(WifiEvent event)
 ///        and emits a WifiEvent::kStationDoDhcp event to trigger DHCP polling checks. Helper function for ProcessEvent.
 void ResetDHCPNotificationFlags(void)
```

Comment:
if DHCP related process is missing do we need this API? if yes, then it should be renamed.